features/refuse_claim.feature

```diff
@@ -23,3 +23,26 @@ Feature: Case worker rejects a claim, providing a reason
     Then the claim I've just updated is no longer in the list
 
     And I eject the VCR cassette
+
+  Scenario: I refuse a claim after a redetermination request providing a reason
+
+    Given a "case worker" user account exists
+    And an "advocate" user account exists
+    And there is a redetermination claim allocated to the case worker with case number 'A20161234'
+
+    And I insert the VCR cassette 'features/case_workers/claims/reject'
+
+    When I am signed in as the case worker
+    And I select the claim
+    And I click the refused radio button
+    And I select the first refusal reason
+    And I click update
+    Then the status at top of page should be Refused
+    Then the messages should not contain 'Total (inc VAT): £0.00'
```

Comment:
Should this be asserting the actual value, rather checking that it's not zero?

The negative check tells us the total amount assessed is not zero, but does not guarantee that it preserves whatever the total amount originally assessed.

**NOTE:** I  guess the only thing that might be a bit trickier for that to work is the fact that the claim is created with some sort of pre-set values that we do not have control in this scenario, so we don't really know what value to expect?

Reply:
This is checking that a message is _not_ displayed.  Previously there would have been the first message stating `Total (inc VAT): £123.00` followed by a message that the claim had been assessed with a total of zero.

This was to ensure that the second message is no-longer displayed